runtime-image/image.yaml

```diff
@@ -38,7 +39,10 @@ envs:
     - name: SSO_DEFAULT_PROVIDER_NAME
       value: rh-sso
 ports:
+    - value: 8443
```

Comment:
do we need to update also line 21 because of this?

Reply:
@jbliznak , was not the case for 7.4 images. I kept the way it was.

Reply:
ok